ios/Classes/core/PHAsset+PM_COMMON.m

```diff
@@ -94,6 +94,41 @@ - (NSString *)title {
         return @"";
     }
 }
+
+- (NSString *)originalFileNameWithSubtype:(int)subtype {
```

Comment:
```suggestion
- (NSString *)originalFilename:(int)subtype {
```

Reply:
It is a general guideline in objective-c to have a method signature with with arguments.